includes/Stories_Renderer/Renderer.php

```diff
@@ -0,0 +1,386 @@
+			return;
+		}
+
+		$web_stories_archive = get_post_type_archive_link( Story_Post_Type::POST_TYPE_SLUG );
+
+		if ( empty( $web_stories_archive ) || ! is_string( $web_stories_archive ) ) {
+			return;
+		}
+
+		?>
+		<div class="web-stories-list__archive-link">
+			<a href="<?php echo( esc_url_raw( $web_stories_archive ) ); ?>">
+				<?php echo( esc_html( $this->attributes['stories_archive_label'] ) ); ?>
+			</a>
+		</div>
+		<?php
+	}
+
+	/**
+	 * Gets the classes for renderer container.
+	 *
+	 * @return string
+	 */
+	protected function get_container_classes() {
+		$container_classes   = [];
+		$container_classes[] = 'web-stories-list';
+		$container_classes[] = ( ! empty( $this->attributes['view_type'] ) ) ? sprintf( 'is-view-type-%1$s', $this->attributes['view_type'] ) : 'is-view-type-circles';
+		$container_classes[] = ( ! empty( $this->attributes['align'] ) ) ? sprintf( 'align%1$s', $this->attributes['align'] ) : 'alignnone';
+		$container_classes[] = ! empty( $this->attributes['class'] ) ? $this->attributes['class'] : '';
+
+		$container_classes = array_filter( $container_classes );
+
+		return implode( ' ', $container_classes );
+	}
+
+	/**
+	 * Gets the single story container classes.
+	 *
+	 * @return string
+	 */
+	protected function get_single_story_classes() {
+
+		$single_story_classes   = [];
+		$single_story_classes[] = 'web-stories-list__story-wrapper';
+		$single_story_classes[] = ! $this->is_view_type( 'grid' ) ? 'has-poster' : '';
+		$single_story_classes[] = $this->is_view_type( 'grid' ) && ! empty( $this->attributes['show_story_poster'] && true === $this->attributes['show_story_poster'] ) ?
+		'has-poster' :
+		'';
+
+		$single_story_classes = array_filter( $single_story_classes );
+
+		/**
+		 * Filters the web stories renderer single story classes.
+		 *
+		 * @param string $class Single story classes.
+		 */
+		return apply_filters( 'web_stories_renderer_single_story_classes', implode( ' ', $single_story_classes ) );
+	}
+
+	/**
+	 * Gets the container style attributes.
+	 *
+	 * @return string
+	 */
+	protected function get_container_styles() {
+		$container_style = ( true === $this->is_view_type( 'grid' ) ) ? sprintf( 'grid-template-columns:repeat(%1$s, 1fr);', $this->attributes['number_of_columns'] ) : '';
+
+		/**
+		 * Filters the web stories renderer container style.
+		 *
+		 * @param string $class Container style.
+		 */
+		return apply_filters( 'web_stories_renderer_container_style', $container_style );
+	}
+
+	/**
+	 * Render story markup.
+	 *
+	 * @param int $story_id Story ID.
+	 *
+	 * @return void
+	 */
+	protected function render_single_story_content( $story_id ) {
+		$story_data           = $this->get_story_item_data( $story_id );
+		$single_story_classes = $this->get_single_story_classes();
+		$show_story_player    = true !== $this->attributes['show_story_poster'] && $this->is_view_type( 'grid' )
```

Comment:
Use a round bracket around the condition. Please update here any everywhere to maintain consistancy.